sda-commons-server-kafka/build.gradle

```diff
@@ -10,21 +10,8 @@ dependencies {
 
   compile ('com.github.ftrossbach:club-topicana-core') {
     exclude group: 'org.slf4j', module: 'slf4j-log4j12'
-    exclude group: 'org.apache.zookeeper', module: 'zookeeper'
     exclude group: 'org.apache.kafka', module: 'kafka_2.12'
     exclude group: 'org.apache.kafka', module: 'kafka_2.11'
-    exclude group: 'org.apache.kafka', module: 'kafka-clients'
-  }
-
-  compile "org.apache.kafka:kafka-clients"
```

Comment:
We should leave this here and don't use the transitive version from clup-topicana?

Reply:
I restored the dependency here as it is actively used in the module.